relay-kafka/src/producer/mod.rs

```diff
@@ -1,3 +1,11 @@
+//! This module contains the kafka producer related code.
+//!
+//! There are two different producers are supported in Relay right now:
+//! - [`SingleProducer`] - which sends all the messages to the defined kafka [`KafkaTopic`]
```

Comment:
```suggestion
//! - [`SingleProducer`] - which sends all the messages to the defined kafka [`KafkaTopic`].
```
Or a coma, not opinionated.